### DISCUSSION TITLE
RTL using <script tag> For Version 2.52

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#2809] [Q&A] [answered]
Disclaimer: I am yet to upgrade due to many customized components built on my side.

Hi

How can I successfully implement RTL functionality when using the script tag? (both daisyui and tailwind)

I understand there is a tailwind plugin I need to use perhaps. 

Can you please advise?

Thanks in advance.

**Comment by saadeghi** [comment, marked as answer]
Documents for version 2.x is here:   
https://v2.daisyui.com/docs/config/#rtl

Before version 4, RTL styles was for build time so you need to enable `rtl` config, then install and add `tailwindcss-flip` and then add `dir='rtl'` to your page.

Let me know if you have any questions.
